src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  get commentButton() {
+    return '[Publish comment] button';
+  }
+
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
```

Comment:
Constructor should modify original `url` by adding `slug` at the end - can you fix this?